org.mixedrealitytoolkit.input/Interactors/Ray/MRTKRayInteractor.cs

```diff
@@ -32,6 +34,47 @@ public class MRTKRayInteractor :
     {
         #region MRTKRayInteractor
 
+        /// <summary>
+        /// Holds a reference to the <see cref="TrackedPoseDriver"/> associated to this interactor if it exists.
+        /// </summary>
+        private TrackedPoseDriver trackedPoseDriver = null;
+
+        /// <summary>
+        /// Property for accessing trackedPoseDriver which holds a reference to the <see cref="TrackedPoseDriver"/> associated to this interactor if it exists.
+        /// </summary>
+        private TrackedPoseDriver TrackedPoseDriver
+        {
+            get
+            {
+                if (trackedPoseDriver == null) //Try to get the TrackedPoseDriver component from the parent if it hasn't been set yet
+                {
+                    trackedPoseDriver = GetComponentInParent<TrackedPoseDriver>();
+                }
+                return trackedPoseDriver;
+            }
+        }
+
+        /// <summary>
+        /// Holds a reference to the <see cref="HandModel"/> associated with this interactor's parent if it exists.
+        /// </summary>
+        private HandModel handModel = null;
+
+        /// <summary>
+        /// The <see cref="HandModel"/> associated with this interactor.  The <see cref="HandModel"/> is obtained from the parent if it hasn't been set yet.
+        /// </summary>
+        private HandModel HandModel
```

Comment:
We don't need a `HandModel`  the data here is already availabe on the base class


```cs
        Handedness IHandedInteractor.Handedness
        {
            get
            {
                #pragma warning disable CS0618 // Type or member is obsolete
                if (forceDeprecatedInput)
                {
                    return (xrController is ArticulatedHandController handController) ? handController.HandNode.ToHandedness() : Handedness.None;
                }
                #pragma warning restore CS0618
                else
                {
                    // TODO Make a helper extension for this before checking in code. See HandednessExtensions.cs for example.
                    return base.handedness == InteractorHandedness.Left ? Handedness.Left : Handedness.Right;
                }
            }
        }
```